lib/matplotlib/tests/test_path.py

```diff
@@ -60,6 +60,22 @@ def test_point_in_path():
     np.testing.assert_equal(ret, [True, False])
 
 
+@pytest.mark.parametrize(
+    "other_path, inside, reverted_inside",
+    [(Path([(0.25, 0.25), (0.25, 0.75), (0.75, 0.75), (0.75, 0.25), (0.25, 0.25)],
+           closed=True), True, False),
+     (Path([(-0.25, -0.25), (-0.25, 1.75), (1.75, 1.75), (1.75, -0.25), (-0.25, -0.25)],
+           closed=True), False, True),
+     (Path([(0.25, 0.25), (0.25, 1.25), (1.25, 1.25), (1.25, 0.25), (0.25, 0.25)],
+           closed=True), False, False),
+     (Path([(0, 0), (0, 1), (1, 1), (1, 0), (0, 0)], closed=True), False, False)]
```

Comment:
Add a fully non-intersecting path as well?

```suggestion
     (Path([(0, 0), (0, 1), (1, 1), (1, 0), (0, 0)], closed=True), False, False),
     (Path([(10, 10), (10, 11), (11, 11), (11, 10), (10, 10)], closed=True), False, False)]    
```